python/nav/web/seeddb/page/management_profile/forms.py

```diff
@@ -30,8 +36,34 @@ class ManagementProfileFilterForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         super(ManagementProfileFilterForm, self).__init__(*args, **kwargs)
-        self.helper = get_formhelper()
-        self.helper.layout = get_single_layout('Filter connection profiles', 'protocol')
+
+        self.attrs = set_flat_form_attributes(
+            form_class="custom",
+            form_fields=[
```

Comment:
```suggestion
        self.attrs = set_flat_form_attributes(
            form_class="custom",
            form_method="get",
            form_fields=[
```
Original form does a GET request, while this new form does a POST request.

Reply:
Good catch! I tested it now and it works again